CONTRIBUTING.md

```diff
@@ -58,42 +61,53 @@ go build
 **Running the linter**:
 
 We make use of the [golangci-lint](https://github.com/golangci/golangci-lint) tool to lint the code in CI. To run it locally, first install it:
+
 ```bash
 go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 ```
+
 then run:
-```
-golangci-lint run --out-format=tab --new-from-rev master ./...
+
+```bash
+make lint
 ```
 
-If you've added new dependencies you might also want to check and make sure all dependencies exists in `vendor/` folder by running:
+You can also run the linter inside the docker container, which will benefit from the version of the linter being the same as it will be in CI.
+
 ```bash
-go get -u github.com/FiloSottile/vendorcheck
-vendorcheck ./...
+make ci-like-lint
 ```
 
 **Running the test suite**:
 
 To exercise the entire test suite:
+
 ```bash
-go test -race ./...
+make test
 ```
 
 To run the tests of a specific package:
+
 ```bash
 go test -race go.k6.io/k6/core
 ```
 
 To run just a specific test case use `-run` and pass in a regex that matches the name of the test:
+
 ```bash
 go test -race ./... -run ^TestEngineRun$
 ```
```

Comment:
I am not particularly certain we need to explain how to run go tests - if the developer isn't familiar with go to know this, I don't think we are the people to teach them that.

Reply:
While I agree that we shouldn't be expected to _teach_ potential contributors how to run tests in Go, we _should_ however lower any barriers for a contributor to add to our project. Maybe instead, we point contributors to make use of the `Makefile` target as a "things to do before submitting a PR." WDYT?

I plan to work up similar for "supported extensions", so this change can be tabled for now and be revisited.

Reply:
Disregard...later convo made me realize there is already reference to `make test` in another section. 🙈